md/bdm-access-control.md

```diff
@@ -5,7 +5,7 @@
 :::
 
 ## Overview
-When defining access control on a business object, one use-case is to define which attributes of this business object a given user is allowed to access through REST API.  
+To Define access control on a business object is to define which attributes of this business object a given user is allowed to access through REST API.  
```

Comment:
Currently, in Bonita, defining access control on a business object means defining (...)